featurebyte/schema/event_table.py

```diff
@@ -23,7 +23,7 @@ class EventTableCreate(TableCreate):
     """
 
     type: Literal[TableDataType.EVENT_TABLE] = TableDataType.EVENT_TABLE
-    event_id_column: StrictStr
+    event_id_column: Optional[StrictStr] = Field(default=None)
```

Comment:
Do we need to create default value for this field?

Reply:
It's not necessary. I added it here for consistency between SDK and REST requests since we provide a default of `None` for `create_event_table` in the SDK (for this function we did not need to change parameter order). Should we also remove the default value there as well if we remove it here?

Reply:
Got it, currently we add default to `None` for all optional field in api payload.